helm-chart/templates/_helpers.tpl

```diff
@@ -107,6 +107,13 @@ Define the default app env variables
                 configMapKeyRef:
                   name: avatar-config
                   key: SHARED_STORAGE_PATH
+            {{- if .Values.api.awsEndpointUrl }}
+            - name: AWS_ENDPOINT_URL
```

Comment:
```suggestion
            - name: S3_URL
```

Reply:
The "official" term is ENDPOINT_URL, whether it be `aws --endpoint-url=...` or `FSSPEC_S3_ENDPOINT_URL` for `fsspec`, so I'd be wary naming is `S3_URL` which might be confused by the the URL of the bucket.

Moreover, modifying it here means I have to re-deploy everything and modify it in the API.

But I am sure this can be improved in the future.

Reply:
OKAY.. If it's "official" then... Why bother, right ? It's sure very appropriate to name something after a service provider we don't use in this case. And besides `FSSPEC_S3_ENDPOINT_URL` should be `FSSPEC_AWS_ENDPOINT_URL` if I follow that logic